my.css

```diff
@@ -352,3 +352,162 @@ body {
     justify-content: center;
   }
 }
+
+/* Вторая секция */
+.bagscluth__content {
+  display: flex;
+  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5642160)